backend/repository/client_update.go

```diff
@@ -46,6 +46,16 @@ type configField struct {
 	ExposedMethods map[string]string `json:"exposedMethods,omitempty"`
 }
 
+func correctMethodNaming(key string) string {
+	for k := range codegen.CommonInitialisms {
+		initial := string(k[0]) + strings.ToLower(k[1:])
+		if strings.Contains(key, string(k[0])+strings.ToLower(k[1:])) {
+			key = strings.Replace(key, initial, k, 1)
```

Comment:
This means `TestUrlAndUrl` will become `TestURLAndUrl`, not `TestURLAndURL. I think you want replace all occurrences.